### PR TITLE
feat: add Requets.reset option 

### PR DESCRIPTION
**Pull request by ronag**

### CODE REVIEW

**Comment by ronag** [comment]
I can't make a test for this since Node server doesn't support keep alive with HEAD requests.

**Review by mcollina — CHANGES_REQUESTED** [review]
Good idea! tests are needed/missing.

File: test/get-head-body.js [hunk 1]
```diff
@@ -182,3 +182,41 @@ test('HEAD should reset connection', (t) => {
...
+//       path: '/',
+//       method: 'HEAD',
+//       reset: false
+//     }, (err, data) => {
+//       t.error(err)
+//       data.body.resume()
+//       done = true
+//     })
+//   })
+// })
```
[the remark this answers is not in the document]
Comment (ronag) [reply test/get-head-body.js:222]: Ideas?

[the remark this answers is not in the document]
Comment (mcollina) [reply test/get-head-body.js:222]: set `reset: false`.

We would also need another tests that do a `GET` with `reset: true`.

**Comment by DarkGL** [comment]
Is this still needed ?

**Comment by ronag** [comment]
Actually this is already implemented.

**Pull request closed without merging**